server/database/postgres.go

```diff
@@ -0,0 +1,238 @@
+package database
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+	"errors"
+	"github.com/jinzhu/gorm"
+//importing Go Postgres driver for database
+	_ "github.com/lib/pq"
+	"log"
+)
+//MUnit represents units of measure used for products
```

Comment:
Please move models to corresponding package or file. Try not to store everything in one file.